test/NameLookup/scope_map_lookup_extension_extension.swift

```diff
@@ -2,5 +2,12 @@
 // RUN: not %target-swift-frontend -typecheck %s 2> %t.errors
 // RUN: %FileCheck %s <%t.errors
 // CHECK-NOT: Program arguments:
+
+// Note: extra newlines below ensure that context printing doesn't show the
```

Comment:
I think this comment would be useful in the other tests that have multiple newlines as well. I was quite confused by them at first.